backstop/scenarios/modal.js

```diff
@@ -6,5 +6,13 @@ module.exports = [
     // when "Open modal" sometimes is and sometimes is not hovered.
     actions: [click('#open-modal'), hover('body')],
     selectors: ['document'],
+    hideSelectors: ['h1', 'hr', 'theme-button', '#open-modal', '#open-full-page-modal'],
```

Comment:
Maybe worth changing to more universal selector: `body > *:not(.iui-modal)`

Reply:
Done